guide/creating-your-bot/README.md

```diff
@@ -139,10 +156,18 @@ client.once('ready', () => {
 // Login to Discord with your client's token
 client.login(token);
 ```
+:::
+::::
 
 This is how you create a client instance for your Discord bot and login to Discord. The `Intents.FLAGS.GUILDS` intents option is necessary for your client to work properly.
 
-Open your terminal and run `node index.js` to start the process. If you see "Ready!" after a few seconds, you're good to go!
+Open your terminal and run:
+
+```sh:no-line-numbers
+node index.js
+``` 
+
+to start the process. If you see "`Ready!`" after a few seconds, you're good to go!
```

Comment:
my thought process when writing this refactor is to put all "actionable" things into code block, instead of in-line.

my definition of "actionable" things is:
- example code (the reader might want to _copy paste_)
- commands in the terminal (the reader must _execute_ the command in their terminal)

if this is not a good enough reason then i can revert it back to in-line code.